cpp/src/arrow/adapters/orc/adapter.cc

```diff
@@ -77,6 +71,15 @@ namespace orc {
   ORC_ASSIGN_OR_THROW_IMPL(ARROW_ASSIGN_OR_RAISE_NAME(_error_or_value, __COUNTER__), \
                            lhs, rexpr);
 
+const uint64_t ORC_NATURAL_WRITE_SIZE = 128 * 1024;  // Required by liborc::Outstream
```

Comment:
nit: fix to use kCamelCase conventions and use constexpr.

Reply:
Thanks! Fixed.